pkg/generator_test.go

```diff
@@ -2306,6 +2305,14 @@ func NewA(t mockConstructorTestingTNewA) *A {
 	s.checkGeneration("struct_value.go", "A", false, "", expected)
 }
 
+func (s *GeneratorSuite) TestGeneratorForStructWithTag() {
+	b, err := os.ReadFile("testdata/struct_with_tag.txt")
```

Comment:
StructTag has backquote, So I can't raw string literals in this test case.

Reply:
Understood, thanks for the explanation, I was going to ask that :)